### ISSUE TITLE
pknViscosityDominated_benchmark does not run

### ISSUE THREAD

**Issue by paveltomin** [GEOS-DEV/GEOS#2826]
File: inputFiles\hydraulicFracturing\pknViscosityDominated_benchmark.xml
Output:
[run.604958.txt](https://github.com/GEOS-DEV/GEOS/files/13395574/run.604958.txt)

**Comment by paveltomin** [comment]
same for the poroelastic version, `pknViscosityDominated_poroelastic_benchmark.xml`, @jhuang2601 do you remember if that was working before?

**Comment by jhuang2601** [comment]
@paveltomin Yes, both cases should be working and last time I checked these cases was this June.

**Comment by paveltomin** [comment]
thanks for confirming, @jhuang2601, looks like I messed up something

**Comment by paveltomin** [comment]
actually, I re-run and ended up with the same convergence failure [run.606509.txt](https://github.com/GEOS-DEV/GEOS/files/13398686/run.606509.txt)
it could be affected by https://github.com/GEOS-DEV/GEOS/pull/2623
CC @CusiniM

**Comment by paveltomin** [comment]
> actually, I re-run and ended up with the same convergence failure [run.606509.txt](https://github.com/GEOS-DEV/GEOS/files/13398686/run.606509.txt) it could be affected by #2623 CC @CusiniM

for poroelastic case - it can finish but cuts time step a lot: [run.606511.txt](https://github.com/GEOS-DEV/GEOS/files/13401382/run.606511.txt)

**Comment by CusiniM** [comment]
> actually, I re-run and ended up with the same convergence failure [run.606509.txt](https://github.com/GEOS-DEV/GEOS/files/13398686/run.606509.txt) it could be affected by #2623 CC @CusiniM

It is possible but it's hard to know.

**Comment by CusiniM** [comment]
> > actually, I re-run and ended up with the same convergence failure [run.606509.txt](https://github.com/GEOS-DEV/GEOS/files/13398686/run.606509.txt) it could be affected by #2623 CC @CusiniM
> 
> for poroelastic case - it can finish but cuts time step a lot: [run.606511.txt](https://github.com/GEOS-DEV/GEOS/files/13401382/run.606511.txt)

Since I have merged https://github.com/GEOS-DEV/GEOS/pull/2757 we can try to set the transversalPermeability and see if the poroelastic case converges better. Also, I have noticed that for poroelasticity there is a change that needs to happen to properly account for the derivative of the  fracture perm w.r.t displacement in the frac-mat connection.

**Comment by paveltomin** [comment]
> > actually, I re-run and ended up with the same convergence failure [run.606509.txt](https://github.com/GEOS-DEV/GEOS/files/13398686/run.606509.txt) it could be affected by #2623 CC @CusiniM
> 
> It is possible but it's hard to know.

I just run the case with version right before that PR: [run.620636.txt](https://github.com/GEOS-DEV/GEOS/files/13420392/run.620636.txt)

And the version right after that PR: [run.621033.txt](https://github.com/GEOS-DEV/GEOS/files/13420397/run.621033.txt)

Clearly it works before and stopped working after. With all good changes in that PR, the effect on the convergence is quite concerning.

**Comment by paveltomin** [comment]
See also https://github.com/GEOS-DEV/GEOS/issues/2853

**Comment by TotoGaz** [comment]
@paveltomin Should we close the duplicates in favor of #2853 ?

**Comment by paveltomin** [comment]
> @paveltomin Should we close the duplicates in favor of #2853 ?

I prefer mine to be opened.
I can guarantee to keep track of the issue I submitted, not sure about the other user.
There are tons of dead-hanging issues and no one cares.

**Comment by jhuang2601** [comment]
@paveltomin I've tested the `pknViscosityDominated_benchmark.xml` with the most recent develop repo and can reproduce the same convergence issue. However, so far, the result is still close to the analytical solution. 
![image](https://github.com/GEOS-DEV/GEOS/assets/53012159/bcaf62ae-6864-475e-9efb-91ff3a0e71ad)
@karimifard @rrsettgast @CusiniM I suspect the reported regression might be related to the way how to handle  `apertureTable` after the update of `ParallelPlatesPermeability` in #2623 or any unexpected change of linear solver?
